src/components/footer/www/footer.jsx

```diff
@@ -219,12 +219,19 @@ const Footer = props => (
 
 Footer.propTypes = {
     intl: intlShape.isRequired,
-    scratchWikiLink: PropTypes.string
+    scratchWikiLink: PropTypes.string,
+    isFullScreen: PropTypes.bool
 };
 
-const mapStateToProps = (state, ownProps) => ({
-    scratchWikiLink: getScratchWikiLink(ownProps.intl.locale)
-});
+const mapStateToProps = (state, ownProps) => {
+    const result = {
```

Comment:
Could you give this variable a more descriptive name? Maybe "props" or "propResults"?